t/lib/OpenQA/Test/Utils.pm

```diff
@@ -607,7 +607,7 @@ sub mock_io_loop (%args) {
 
 sub assume_all_assets_exist { OpenQA::Schema->singleton->resultset('Assets')->search({})->update({size => 0}) }
 
-sub schedule_iso ($t, $args, $status = 200, $query_params = {}, $msg = undef) {
```

Comment:
You're adding a new parameter here but it isn't used. So I suggest you leave that change out.